### ISSUE TITLE
ios build problems

### ISSUE THREAD

**Issue by bartschuller** [typester/auth2#3]
Hi,

Thanks for the work you're putting into this. I'm following along with your tutorial and I encountered two problems, one of which I could fix.

The `cargo-lipo` step failed with the following on an M4 Mac:
```
error[E0463]: can't find crate for `core`
  |
  = note: the `x86_64-apple-ios` target may not be installed
  = help: consider downloading the target with `rustup target add x86_64-apple-ios`
```
so that was easy to fix. But then when these two commands finished succesfully:
```
cargo lipo --release
cargo build --release --target aarch64-apple-ios-sim
```
I couldn't open the ios project in XCode because `auth2FFI.h`, referenced from https://github.com/typester/auth2/blob/main/ios/Auth2/Bridging-Header.h#L8 is nowhere to be found.

Shouldn't there be a call to `uniffi-bindgen` somehwere for ios too?

**Comment by typester** [comment]
Thank you for your feedback!

You're absolutely right, that was my bad. The step to generate bindings for iOS was missing.

You’ll need to generate the Swift glue code with the following command:

```
$ cargo run --features=uniffi-cli --bin uniffi-bindgen generate -l swift -o ../ios/Auth2/uniffi/ --library target/aarch64-apple-ios/release/libauth2.a
```

Could you try running this and see if it resolves the issue? I'll make sure to update the blog post to include this step soon. Thanks again for bringing this to my attention!

**Comment by bartschuller** [comment]
Yes, after running that in `core` I can build and run the XCode project. The simulator starts and the app works!

There's just a warning after running `uniffi-bindgen`:

```Warning: Unable to auto-format auth2.swift using swiftformat: Os { code: 2, kind: NotFound, message: "No such file or directory" }```

but that can be ignored it seems.

Thank you!
